src/ci/resources/dockerhub_provider_config_v2_google_secret_manager_usa.inc.conf.ctmpl

```diff
@@ -0,0 +1,11 @@
+{{with $cromwellDockerhub := secret (printf "secret/dsde/cromwell/common/cromwell-dockerhub")}}
+dockerhub {
+  # GCP Batch supports both the usual base64-encoded <username>:<password> Docker token format,
+  # but also supports the use of Google Secret Manager (GSM). When using GSM the token is still base64
+  # encoded but the format becomes
+  # <Path to GSM username secret>:<Path to GSM password secret>
+  # This test is exercising the GCP Batch Google Secret Manager support.
+  token = "{{$cromwellDockerhub.Data.google_secret_manager_token}}"
```

Comment:
Would you mind also creating a ticket to cover the changes we'll need to make to get this working in Terra? Currently we store a `docker_token` that is b64 encoded `username:password`, we'll need to add a secret containing just the password and maybe another containing the new token, and update Cromwell config to use the new secret.

Reply:
https://broadworkbench.atlassian.net/browse/AN-211